src/client/pages/Offer/Checkout/CheckoutSuccessRedirect.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react'
```

Comment:
Here, we ended up doing only two things:
* Replacing the old `useCurrentLocale` with the new one
* Slightly changed the type definition of offerData and added line 19
For he readebility of this PR, we should maybe put the changes we made to this file in the older `OfferNew` file and point to that one.